src/Symfony/Component/Routing/Tests/Fixtures/dumper/url_matcher5.php

```diff
@@ -21,185 +21,110 @@ public function match($rawPathinfo)
-            // nested_a
-            if ('/nested/group/a' === $trimmedPathinfo) {
-                $ret = array('_route' => 'nested_a');
-                if ('/' === substr($pathinfo, -1)) {
-                    // no-op
-                } elseif ('GET' !== $canonicalMethod) {
-                    goto not_nested_a;
-                } else {
-                    return array_replace($ret, $this->redirect($rawPathinfo.'/', 'nested_a'));
+                if ($requiredMethods && !isset($requiredMethods[$canonicalMethod]) && !isset($requiredMethods[$requestMethod])) {
+                    $allow += $requiredMethods;
+                    break;
                 }
 
                 return $ret;
-            }
-            not_nested_a:
-
-            // nested_b
-            if ('/nested/group/b' === $trimmedPathinfo) {
-                $ret = array('_route' => 'nested_b');
-                if ('/' === substr($pathinfo, -1)) {
-                    // no-op
-                } elseif ('GET' !== $canonicalMethod) {
-                    goto not_nested_b;
-                } else {
-                    return array_replace($ret, $this->redirect($rawPathinfo.'/', 'nested_b'));
-                }
-
-                return $ret;
-            }
-            not_nested_b:
-
-            // nested_c
-            if ('/nested/group/c' === $trimmedPathinfo) {
-                $ret = array('_route' => 'nested_c');
-                if ('/' === substr($pathinfo, -1)) {
-                    // no-op
-                } elseif ('GET' !== $canonicalMethod) {
-                    goto not_nested_c;
-                } else {
-                    return array_replace($ret, $this->redirect($rawPathinfo.'/', 'nested_c'));
-                }
-
-                return $ret;
-            }
-            not_nested_c:
-
         }
 
-        elseif (0 === strpos($pathinfo, '/slashed/group')) {
-            // slashed_a
-            if ('/slashed/group' === $trimmedPathinfo) {
-                $ret = array('_route' => 'slashed_a');
-                if ('/' === substr($pathinfo, -1)) {
-                    // no-op
-                } elseif ('GET' !== $canonicalMethod) {
-                    goto not_slashed_a;
-                } else {
-                    return array_replace($ret, $this->redirect($rawPathinfo.'/', 'slashed_a'));
-                }
-
-                return $ret;
-            }
-            not_slashed_a:
-
-            // slashed_b
-            if ('/slashed/group/b' === $trimmedPathinfo) {
-                $ret = array('_route' => 'slashed_b');
-                if ('/' === substr($pathinfo, -1)) {
-                    // no-op
-                } elseif ('GET' !== $canonicalMethod) {
-                    goto not_slashed_b;
-                } else {
-                    return array_replace($ret, $this->redirect($rawPathinfo.'/', 'slashed_b'));
+        $matchedPathinfo = $pathinfo;
+        $regexList = array(
```

Comment:
We don't have a test case where `$regexList` contains more than one regex, do we? This seems missing.

Reply:
There was a functional one. I just added a dumped one also.